### PR TITLE
Multistore - Catalog > Products > Add / edit - Update message when a group is selected

### LINKED ISSUE #19459: Multistore - Catalog > Products > Add / edit - Update message when a group is selected
Currently, this page isn't working in a group context
The current information message is "You can't add or edit a product in this shop context: select a shop instead of a group of shops." 

For a better understanding, we should update the current message for the following one:
"Note that this page is available in a single shop or in all shops context. Switch context to work on it."

This is specified in the [multistore specials cases specs](https://github.com/PrestaShop/prestashop-specs/blob/master/broader-topics/multistorespecialsspecs.md#list-of-pages-for-specific-case-6-the-page-is-displayed-only-when-a-specific-shop-or-all-shops-is-selected)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Multistore - Catalog > Products > Add / edit - Update message when a group is selected
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #19459
| How to test?      | Cf. #19459

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27680)

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php [hunk 1]
```diff
@@ -513,12 +513,8 @@ private function renderDisableMultistorePage(int $productId = null): Response
     {
         return $this->render('@PrestaShop/Admin/Sell/Catalog/Product/disabled.html.twig', [
             'errorMessage' => $this->trans(
-                'This page is only compatible in a single store context. To access the page, please select a store or [1]disable the multistore feature[/1].',
-                'Admin.Notifications.Info',
-                [
-                    '[1]' => sprintf('<a href="%s">', $this->get('router')->generate('admin_preferences')),
-                    '[/1]' => '</a>',
-                ]
+                'Note that this page is available in a single shop or in all shops context. Switch context to work on it.',
```
Comment (Julievrz) [inline src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php]:
```suggestion
                'This page isn't available when working on a group of stores. Switch context to work on it.',
```

**Review by Julievrz — COMMENTED** [review]
A wording suggestion

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php (hunk 1 above)
Comment (Progi1984) [reply src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php]: Fixed @Julievrz

**Review by Julievrz — APPROVED** [review]
Thanks, wording ✔️

**Review by saulaski — APPROVED** [review]

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php [hunk 2]
```diff
@@ -471,12 +471,8 @@ private function renderDisableMultistorePage(int $productId = null): Response
     {
         return $this->render('@PrestaShop/Admin/Sell/Catalog/Product/disabled.html.twig', [
             'errorMessage' => $this->trans(
-                'This page is only compatible in a single store context. To access the page, please select a store or [1]disable the multistore feature[/1].',
-                'Admin.Notifications.Info',
-                [
-                    '[1]' => sprintf('<a href="%s">', $this->get('router')->generate('admin_preferences')),
-                    '[/1]' => '</a>',
-                ]
+                'This page isn\'t available when working on a group of stores. Switch context to work on it.',
```
Comment (jolelievre) [inline src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php:474]: The issue you are referring to is quite old It was related to legacy page
This is v2 page and the current wording has been validated by @Julievrz so I think we should close this PR but I let her validate this

**Review by jolelievre — CHANGES_REQUESTED** [review]
Sorry @Progi1984 but I don't think this modification is needed anymore

**Comment by Progi1984** [comment]
@Julievrz Could you look at the @jolelievre comment, please ?

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php (hunk 2 above)
Comment (Julievrz) [reply src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php:474]: @Progi1984 So, as discussed with Jonathan, the right wording is "This page is only compatible in a single store context. To access the page, please select a store or [1]disable the multistore feature[/1]."

I thought the behavior had changed again, sorry. 😅

Comment (Progi1984) [reply src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php:474]: No problem. I close the PR and the issue.

**Pull request closed without merging**